js/components/open-list-nav.js

```diff
@@ -0,0 +1,114 @@
+(function(exports) {
+    function getScrollTop() {
+      return scrollTop = (window.pageYOffset !== undefined)
```

Comment:
Did you mean to return a conditional instead of an assignment?<br>'scrollTop' is not defined.